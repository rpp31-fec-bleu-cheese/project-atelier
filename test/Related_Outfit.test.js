

// import React from 'react';
// import Enzyme from 'enzyme';
// import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
// import 'jsdom-global/register';
// import sinon from 'sinon';
// import jest from 'jest';
// import Related_Outfit from '../client/src/components/Related_Outfit/Related_Outfit.jsx';


// Enzyme.configure({adapter:new Adapter()});


// describe('Related_Outfit', () => {
//   //test to see if Related_Outfit component is rendered
//   it('Related_Outfit should render correctly', () => {
//     const wrapper = Enzyme.mount(<Related_Outfit/>);
//     expect(wrapper).toMatchSnapshot();

//    // expect(Foo.prototype.componentDidMount).to.have.property('callCount', 1);
//   });
//   it('componentDidMount() is called', (done) => {
//     const wrapper = Enzyme.mount(<Related_Outfit/>);
//     const instance = wrapper.instance();
//     sinon.spy(instance, 'fetchRelatedInfo');
//     instance.componentDidMount();
//     console.log('instance:', instance.fetchRelatedInfo);

//     expect(instance.fetchRelatedInfo).toHaveBeenCalledTimes(1);
//     done();
//    // expect(Foo.prototype.componentDidMount).to.have.property('callCount', 1);
//   });
//   /*it('calls componentDidMount', () => {
//     const wrapper = Enzyme.mount(<Related_Outfit/>);
//     sinon.spy(Related_Outfit.prototype,'componentDidMount');

//     //expect(Related_Outfit.prototype.componentDidMount).to.have.property('callCount', 1);
//   });*/
//   /*it('Related_ProductInfo should render', () => {
//     const wrapper = Enzyme.mount(<Related_Outfit/>);
//     //console.log(wrapper.find(Related_ProductInfo).debug())
//     expect(wrapper.matchesElement(<button className="NextProd Related"></button>)).to.equal(true);
//     expect(wrapper.find('Related_ProductInfo')).to.equal(true);
//   })*/

// });