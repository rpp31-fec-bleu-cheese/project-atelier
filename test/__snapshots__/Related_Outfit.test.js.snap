// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Related_Outfit Related_Outfit should render correctly 1`] = `ReactWrapper {}`;

exports[`Related_Outfit should render correctly 1`] = `ReactWrapper {}`;

exports[`Related_Outfit should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
